apps/roam/src/utils/initializeObserversAndListeners.ts

```diff
@@ -115,8 +125,12 @@ export const initObservers = async ({
     callback: (s: HTMLSpanElement) => {
       const tag = s.getAttribute("data-tag");
       if (tag) {
-        if (discourseTagSet.has(tag.toLowerCase())) {
+        const style = discourseTagToStyle[tag.toLowerCase()];
+        if (style) {
           renderNodeTagPopupButton(s, discourseNodes, onloadArgs.extensionAPI);
+          s.style.color = style.color;
```

Comment:
Could we put this color/style directly on the Blueprint button?  That way it would all be in one place.  Also, we could use tailwind/classes as much as possible so users could adjust the styles themselves.

Reply:
sorry which button? We have to add the color the node tag in the block right?

<img width="746" height="198" alt="image" src="https://github.com/user-attachments/assets/50552194-9cb2-42fa-a0d4-e79f9384abaf" />

Reply:
Ah right, my bad.  I was thinking of the popover.